compose/foundation/foundation/src/desktopMain/kotlin/androidx/compose/foundation/Scrollbar.desktop.kt

```diff
@@ -544,40 +538,45 @@ private class SliderAdapter(
             return if (extraContentSpace == 0f) 1f else extraScrollbarSpace / extraContentSpace
         }
 
-    /**
-     * A position with cumulative offset, may be out of the container when dragging
-     */
```

Comment:
I think this comment may help with better understanding.
Is this logic still correct? (If yes - then better to keep it)

Reply:
No, this comment and the logic of using it as the cumulative offset was the part of the original commit that I removed.
The cumulative offset is now `positionDuringDrag`